app/autofill.js

```diff
@@ -0,0 +1,14 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const appActions = require('../js/actions/appActions')
+
+module.exports.init = () => {
+  process.on('personal-data-changed', (profileGuids, creditCardGuids) => {
+    setImmediate(() => {
+      appActions.updateAutofillAddress(profileGuids)
```

Comment:
can we combine these into a single `autofillDataChanged` action? Every emit from the appStore causes another render cycle for a lot of components and it also follows the declarative action pattern better